### DISCUSSION TITLE
Load Data into Table Plugin on Demand

### DISCUSSION THREAD

**Discussion by ghost** [grafana/grafana#63514] [Plugins]
I am using table plugin to show data of 10000 Rows, i.e. query gets data of all the 10000 rows which is taking time to complete.
Is there a way to show pagination or Infinite scroll where query is fired on demand with limit and offset.

So lets set limit to 20 when i scroll down query is fired and adds 20 rows to existing 20 rows.

**Comment by Sam-H101** [comment]
that would be a cool feature to be added, It may be easier to implement using pages then scrolling.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82838. Feel free to continue the discussion around this there. Thank you!
